public/editor/scripts/ui/menus.js

```diff
@@ -126,10 +126,17 @@ function setupOptionsMenu(bramble) {
     "#editor-pane-nav-options-menu"
   );
 
-  // Font size
+  // Display font size
+  $(document).ready(function(){
```

Comment:
The Travis build is complaining about this line:

```
/home/travis/build/mozilla/thimble.mozilla.org/public/editor/scripts/ui/menus.js
  130:31  error  Insert `·`  prettier/prettier
```

@gideonthomas what is the issue here?

Reply:
@azusaaz for future, when you have an automated build fail like this, you can click the `Details` link in your PR to have it show you what's wrong:

https://travis-ci.org/mozilla/thimble.mozilla.org/builds/289327030#L483

Reply:
Also, the AppVeyor build is failing with the same problem, so fixing it once will fix both builds:

https://ci.appveyor.com/project/mozillafoundation/thimble-mozilla-org/build/77#L1113

Reply:
@humphd  Thank you! I'll check the line.